components/CheckDocumentation/connections/BlockingQuery.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+
+import SQL from "../../SQL";
+
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const BlockingQueryTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  return (
+    <>
+      <p>
+        Detects queries currently blocking equal or more than the specified
+        threshold of <code>{config.settings["blocked_count"]}</code> queries.
```

Comment:
I see what you mean about the wording being more awkward here with `>=`. Maybe something like

```
        Detects queries currently blocking at least <code>{config.settings["blocked_count"]}</code>
        other queries, according to the specified threshold.
```
? Or, looking at what we do for `system/storage_space`, we could just reference the setting directly, and let the user infer that this is configurable:

```
        Detects queries currently blocking at least <code>{config.settings["blocked_count"]}</code>
        other queries.
```

I think if you read this and these numbers don't work for you, checking the config page to see if you can change that seems pretty natural, especially since this is shown in the "Check Configuration" panel, with a "Configure this check" link at the bottom...